pkgs/by-name/ne/nextjs-ollama-llm-ui/package.nix

```diff
@@ -0,0 +1,94 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  inter,
+  lib,
+  # This is a app can only be used in a browser and starts a web server only accessible at
+  # localhost/127.0.0.1 from the local computer at the given port.
+  defaultHostname ? "127.0.0.1",
+  defaultPort ? 3000,
+  # Where to find the Ollama service; this url gets baked into the Nix package
+  ollamaUrl ? "http://127.0.0.1:11434",
+  ...
+}:
+
+let
+  version = "1.0.1";
+in
+buildNpmPackage {
```

Comment:
Last thing, you can add in passthru your new nixosTest and we're done.

Reply:
Done. Although i have no clue how that value is used or can/should be tested. Can you recommend a checklist/guide on all the things people should think about to avoid this back-and-forth?

Reply:
No I don't but it would be super great to do. Thanks for your patience and for all the changes, I don't have anything else to add here.